starlite/testing/client/base.py

```diff
@@ -104,3 +118,84 @@ def portal(self) -> Generator["BlockingPortal", None, None]:
     def _create_session_cookies(backend: "CookieBackend", data: Dict[str, Any]) -> Dict[str, str]:
         encoded_data = backend.dump_data(data=data)
         return {cookie.key: cast("str", cookie.value) for cookie in backend._create_session_cookies(encoded_data)}
+
+    async def set_session_data_async(self, data: Dict[str, Any]) -> None:
```

Comment:
```suggestion
    async def _set_session_data(self, data: Dict[str, Any]) -> None:
```

These should be private methods